README.md

```diff
@@ -36,6 +36,10 @@ The following types of data are being considered:
  | URL         | Url with more information on the image|
  | Summary     | Short Description of the image|
  | Description | Detailed description of the image|
+ | VcsType    | Where the image source can be obtained from a version control system, which software is used|
```

Comment:
I've just pushed a change that is a variation on @philips suggestion. I've reworded slightly to avoid ambiguity between the container source and the application within (I'm thinking of a situation where the application inside does something with VCS; perhaps a container of gitlab; or the distinction between the source for the app, and the source for the container of the app).

I re-ordered the suggested VCS types in order of 'recommendation' (demoting CVS). I agree that the contents of the label will have to be standardised to be useful; hopefully this won't be a big problem in practise (git == git, at least :). If anyone wants to add or use another VCS, they could always submit another PR.